info/buildSrc/src/main/kotlin/org/cqfn/diktat/generation/docs/GenerationDocs.kt

```diff
@@ -131,6 +133,27 @@ enum class FindType {
     ;
 }
 
+private fun handleHyperlinks(line: String): String {
+    var correctedString = line
+    if (correctedString.contains(HYPERLINKS)) {
+        val hyperlinkSubString = HYPERLINKS.find(correctedString)!!.groupValues
+        hyperlinkSubString.forEach {
+            var link = Regex("""\(http.*\)""").find(it)!!.groupValues.first().drop(1).dropLast(1) // drop ( and )
+            // need to replace back, because it is a hyperlink
+            link = link.replace("\\#", "#")
+            link = link.replace("\\&", "&")
+            link = link.replace("\\_", "_")
+            var text = Regex("""\[.*]""").find(it)!!.groupValues.first()
+            // need to replace ` in hyperlink, because it breaks latex compilation
+            text = text.replace("`", "")
```

Comment:
Maybe, replace with quotes that don't break latex? Do `'` fit here?

Reply:
Backticks make words highlighting in markdown.  Can we delete them here, because it will be highlighted as hyperlink?

Reply:
Backticks can highlight code inside link text [like `this`](https://example.com). Do we really have such cases in our text? Aren't all code snippets already converted to `texttt` in your converter? 
I think it should be safe to remove them here, though.